doc/rados/operations/monitoring.rst

```diff
@@ -130,7 +130,7 @@ Monitoring Health Checks
 
 Ceph continously runs various *health checks* against its own status.  When
 a health check fails, this is reflected in the output of ``ceph status`` (or
-``ceph health``).  In addition, messages are sent to the cluster log to
+``ceph health``).  In addition:, messages are sent to the cluster log to
```

Comment:
That colon after "addition" looks misplaced. Maybe better: "Additionally, messages are sent..."?